### DISCUSSION TITLE
Improvement ideas for services page

### DISCUSSION THREAD

**Discussion by arminus** [SquirrelCorporation/SquirrelServersManager#36] [Ideas]
Some ideas for the docker services page (it will get difficult to manage with > 9 services quickly due to the current paging only approach):

- filter by device, update status, ...
- search for service name
- add an optional table layout
- hook up docker commands (pull/update, stop, start, restart)
- add container logs view

**Comment by SquirrelDeveloper** [comment]
Filters didnt worked until recently, the build version is having some options available for filters!
Actions are underway :-)

**Comment by SquirrelDeveloper** [comment]
Most of those suggestions were implemented, or backlogged. Enjoy!
